src/components/header/header.module.css

```diff
@@ -137,8 +136,9 @@
 }
 
 @media screen and (min-width: 768px) {
-  .logoLink svg.greenwood-logo-full {
-    width: 60%;
+  .navBar {
```

Comment:
I might consider sizing it down a "little" bit between the 768px - 1024px range, as it does seem to dominate a bit of the header in that gap.
<img width="1006" alt="Screenshot 2024-10-15 at 4 12 18 PM" src="https://github.com/user-attachments/assets/c73e2273-7027-40e9-bfb3-296fd3e5a9b0">


Maybe we can `clamp` it or put a `max-width` or something on it?